codelists/tests/test_presenters.py

```diff
@@ -124,59 +123,53 @@ def test_tree_tables():
     #   └  239964003 (Soft tissue lesion of elbow region)
 
     assert presenters.tree_tables(
-        ancestor_codes, hierarchy, code_to_term, code_to_type, code_to_status
+        ancestor_codes, hierarchy, code_to_term, code_to_type
     ) == [
         {
             "heading": "Disorder",
+            # fmt: off
```

Comment:
Is black collapsing the dictionaries into single lines?

Reply:
It's doing it for some lines and not for others

```
            "rows": [
                {"code": "128133004", "term": "Disorder of elbow", "pipes": [],},
                {"code": "429554009", "term": "Arthropathy of elbow", "pipes": ["├"],},
                {
                    "code": "439656005",
                    "term": "Arthritis of elbow",
                    "pipes": ["│", "└"],
                },
                {
                    "code": "202855006",
                    "term": "Lateral epicondylitis",
                    "pipes": ["│", " ", "└"],
                },
                {
                    "code": "35185008",
                    "term": "Enthesopathy of elbow region",
                    "pipes": ["├"],
                },
                {"code": "73583000", "term": "Epicondylitis", "pipes": ["│", "└"],},
                {
                    "code": "202855006",
                    "term": "Lateral epicondylitis",
                    "pipes": ["│", " ", "└"],
                },
                {
                    "code": "239964003",
                    "term": "Soft tissue lesion of elbow region",
                    "pipes": ["└"],
                },
            ],
```

Reply:
The second to last comma on this line:

```{"code": "128133004", "term": "Disorder of elbow", "pipes": [],},```

_should_ make Black split up the items per line (or rather, leave it how you wrote it!), do you have black==20.8b1 installed?  It was supposed to fix this kind of error!